pkg/remote/README.md

```diff
@@ -0,0 +1,59 @@
+# remote: push arbitrary trace level data to an influxdb instance
+
+This package has code to create a client that can be used to push events to an
+influxdb instance. It is used to collect trace data from many different nodes in
+a network. If there is no URL in the config.toml, then the underlying client is
+nil and no points will be written. The provided chainID and nodeID are used to
+tag all points. The underlying client is exposed to allow for custom writes, but
+the WritePoint method should be used for most cases, as it enforces the schema.
+
+## Usage and Schema
+
+To use this package, first create a new client using the `NewClient` function,
+then pass that client to the relevant components that need to push events. After
+that, you can use the `WritePoint` method to push events to influxdb. In the below
+example, we're pushing a point in the consensus reactor to measure exactly when
+each step of consensus is reached for each node.
+
+```go
+if cs.eventCollector.IsCollecting() {
+		cs.eventCollector.WritePoint("consensus", map[string]interface{}{
+			"roundData": []interface{}{rs.Height, rs.Round, rs.Step},
+		})
+}
+```
```

Comment:
added this `IsCollecting` method so that if we do eventually add default points of data collection, then we can avoid allocating memory for structs that we instantly throw away.